test/telemetry/src/main/java/org/opensearch/test/telemetry/tracing/validators/TotalParentSpansEqualToRequests.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.test.telemetry.tracing.validators;
+
+import org.opensearch.test.telemetry.tracing.MockSpanData;
+import org.opensearch.test.telemetry.tracing.TracingValidator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * TotalParentSpansEqualToRequests validator to check sanity on total parent spans.
+ */
+public class TotalParentSpansEqualToRequests implements TracingValidator {
```

Comment:
is it TotalRootSpanEqualToRequests?

Reply:
Renamed it to TotalRootSpansEqualToRequests